weavetest/assert/assert.go

```diff
@@ -105,3 +105,21 @@ func FieldError(t testing.TB, err error, fieldName string, want *errors.Error) {
 
 	}
 }
+
+// IsErr is a convenient helper that checks if the errors are a match
+// and prints out the difference if not as well as failing the assertion.
+func IsErr(t testing.TB, want, got error) {
```

Comment:
I would love to see a simple test on this function. I guess you would need to pass in a fake `testing.TB` interface though. Hmmm... I will make another PR for that maybe

Reply:
We have [`tmock`](https://github.com/iov-one/weave/blob/master/weavetest/assert/assert_test.go#L56) defined. It can be used for those tests.